hedera-node/hedera-consensus-service-impl/src/main/java/com/hedera/node/app/service/consensus/impl/WritableTopicStore.java

```diff
@@ -59,7 +59,7 @@ public WritableTopicStore(@NonNull final WritableStates states) {
      * @param topic - the topic to be mapped onto a new {@link MerkleTopic} and persisted.
      */
     public void put(@NonNull final Topic topic) {
-        Objects.requireNonNull(topicState).put(EntityNum.fromLong(topic.topicNumber()), Objects.requireNonNull(topic));
+        Objects.requireNonNull(topicState).put(requireNonNull(topic.id()), Objects.requireNonNull(topic));
```

Comment:
Should not wrap a class attribute in `Objects.requireNonNull`.  That method is for checking method parameters, not class attributes.  The attribute was already checked in the constructor.

Reply:
ok fixed it

Reply:
I should have been clearer.  The class attribute `topicState` is never null, so it should not be wrapped in `Objects.requireNonNull`.
I did not mean to change the call to `topic.id()` (although there, it's not necessary to wrap `topic` twice in the same line, the first call will fail if it is null).

Reply:
issue #7218 opened for it.

Reply:
The issue applies to a different comment.
This one is still unresolved as you need to remove the unnecessary call in this class for the class attribute (which has nothing whatsoever to do with the issue opened).